Competition_Task/Competition_Task/Tests/CertificationsTests.cs

```diff
@@ -0,0 +1,224 @@
+﻿using Competition_Task.Pages;
+using Competition_Task.TestData;
+using Competition_Task.Utilities;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+
+namespace Competition_Task.Tests
+{
+    [TestFixture]
+    
+    public class CertificationsTests:BaseClass
+    {       
+
+            LoginPage LoginPageObj = new LoginPage();
+            HomePage HomePageObj = new HomePage();
+            CertificationsTab CertificationsTabObj = new CertificationsTab();
+
+
+            public static string certificatedata = File.ReadAllText(@"A:\Industry Connect\MarsCompetition_Task\Competition_Task\Competition_Task\TestData\Certifications.json");
```

Comment:
this can be moved into test and read within the test

Reply:
moved into the Test 
![image](https://github.com/Shilpii01/MarsCompetition_Task/assets/145233624/d48b7a03-11f9-41e7-a588-08692c3600c6)